exercicios/exercicio3.js

```diff
@@ -1 +1,7 @@
+// 3. Liste uma lista de mercado
+
 const listaDeMercado = ['maçã', 'banana', 'chamyto', 'nutella', 'coca-cola', 'bisnaguinha', 'rap10']
+
+const constroiLista = (arr) => {for (i = 0; i < arr.length; i++){console.log(`item ${i+1}: ${arr[i]}`)}}
```

Comment:
Aqui vale a mesma dica pra quebrar o `for` em várias linhas! Além disso, que tal tentar obter esse mesmo resultado, mas usando uma outra estrutura de loop? (: